src/sparseml/pytorch/models/classification/resnet.py

```diff
@@ -140,26 +140,17 @@ def required(in_channels: int, out_channels: int, stride: int) -> bool:
         return in_channels != out_channels or stride > 1
 
 
-class _AddReLU(Module):
+class _NoOp(Module):
```

Comment:
we might want a more descriptive input here since this will be printed out as part of the model definition. maybe `_AddInput`

Reply:
Good suggestion